package.json

```diff
@@ -270,6 +270,11 @@
             "type": "boolean",
             "default": false,
             "description": "Disable telemetry data collection of this Azurite execution. By default, Azurite will collect telemetry data to help improve the product."
+          },
+          "azurite.blobVersioning": {
```

Comment:
As we discussed ,this parameter is temperate for develop/test. 
We need time to decided how to input the Storage management plane config to Azurite.